CONTRIBUTING.md

```diff
@@ -118,6 +119,51 @@ urlpatterns = patterns(
 )
 ```
 
+##### Naming conventions
+Avoid names which does not give additional meaning or are unclear. Some
```

Comment:
I'm missing the reference to [PEP8 Prescriptive naming conventions](https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions).